localisation/00_wc_artifact_localisation.csv

```diff
@@ -1,22 +1,29 @@
+aluneth_desc;This mysterious staff is a source of great magic. Although Aluneth's origin is unknown, the civilization that created it had a very deep knowledge of arcane magic.;;Dieser mysteri�se Stab ist eine Quelle gro�er Magie. Obwohl Aluneths Herkunft unbekannt ist, hatte die Zivilisation, die sie schuf, ein profundes Wissen �ber arkane Magie.;;;;;;;;;;;x
 apocalypse;Apocalypse;;;;;;;;;;;;;x
-apocalypse_desc;Vampiric demons known as nathrezim forged this ancient sword, which brings with it violence, plagues, and death. Apocalypse soon earned a horrific reputation in the hands of a Tirisgarde mage who couldn't control its destructive energies.;;;;;;;;;;;;;x
+apocalypse_desc;The vampiric Nathrezim forged this sword that brings violence, sickness and death with it. Apocalypse gained a terrible reputation in the hands of a Tirisguard mage who could not control the destructive energies.;;Die vampirischen Nathrezim schmiedeten dieses Schwert, das Gewalt, Krankheit und Tod mit sich bringt. Apokalypse erwarb sich einen schrecklichen Ruf in den H�nden eines Tirisgarde-Magiers, der die zerst�rerischen Energien nicht kontrollieren konnte.;;;;;;;;;;;x
 ashbringer;Ashbringer;;;;;;;;;;;;;x
-ashbringer_desc;This sword has been forged from the crystal of pure light. Faced with it, any evil being will be turned into ashes by its purifying light.;;Dieses Schwert wurde aus einem Kristall aus reinem Licht geschmiedet. Angesichts dessen wird jedes �ble Wesen durch sein reinigendes Licht zu Asche zerfallen.;;;;;;;;;;;
+ashbringer_desc;This sword has been forged from the crystal of pure light. Faced with it, any evil being will be turned into ashes by its purifying light.;;Dieses Schwert wurde aus dem Kristall des reinen Lichts geschmiedet. Angesichts dessen wird jedes b�se Wesen durch sein reinigendes Licht in Asche verwandelt.;;;;;;;;;;;x
 atiesh;Atiesh;;;;;;;;;;;;;x
-atiesh_desc;Atiesh, the greatstaff of the Guardian is a powerful staff passed down through the line of the Guardians of Tirisfal. The staff was born from the �seed of hate� and grants its wielder �unending power�. It was passed down by each Guardian of Tirisfal to its successor.;;;;;;;;;;;;;x
+atiesh_desc;Atiesh, the greatstaff of the Guardian is a powerful staff passed down through the line of the Guardians of Tirisfal. The staff was born from the �seed of hate� and grants its wielder �unending power�.;;Atiesh, der Langstab des W�chter ist ein m�chtiger Stab, der von einem W�chter von Tirisfal an den n�chsten weitergegeben wird. Der Stab wurde aus dem �Samen des Hasses� geboren und gew�hrt seinem Besitzer �unendliche Macht�.;;;;;;;;;;;x
 book_of_medivh;Book of Medivh;Livre de Medivh;Medivhs Zauberbuch;;;;;;;;;;;x
-book_of_medivh_desc;The spellbook written by Medivh.;;;;;;;;;;;;;x
+book_of_medivh_desc;The spellbook written by Medivh.;Le livre envo�tant �crit par Medivh.;Das Zauberbuch von Medivh.;;;;;;;;;;;x
```

Comment:
Just adding flavor relating to events in actual lore, while not violating the timeframe of the mod.
```suggestion
book_of_medivh_desc;The spellbook written by Medivh. Truly an artifact of great magical power, particularly when it comes to portal and teleportation magic.;Le livre envo�tant �crit par Medivh.;Das Zauberbuch von Medivh.;;;;;;;;;;;x
```

Reply:
done